app/database.py

```diff
@@ -10,10 +11,15 @@
     echo=log_settings.db_engine_echo,
 )
 
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
 
-def get_session():
-    with Session(engine) as session:
-        yield session
+Base = declarative_base()
 
+def get_session():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
```

Comment:
```suggestion
with SessionLocal() as db:
    yield db
```

Same here as @SamuelSlavik mentioned. Context manager will handle all exceptions in with block, close the db and pass them higher.